triple-store/triple-store-api/src/main/java/com/powsybl/triplestore/api/PropertyBags.java

```diff
@@ -115,14 +115,12 @@ private String tabulate(BiFunction<PropertyBag, String, String> getValue) {
         String columnSeparator = " \t ";
         String lineSeparator = System.lineSeparator();
 
-        StringBuilder s = new StringBuilder(size() * 80);
```

Comment:
I think it would be wiser to revert this change to.
It may also impact the performances (same as previous comment).